clippy_lints/src/borrow_deref_ref.rs

```diff
@@ -42,7 +42,7 @@ declare_clippy_lint! {
     /// # fn foo(_x: &str) {}
     /// # let s = &String::new();
     /// let a: &String = s;
-    /// foo(&**s);
+    /// foo(&*s);
```

Comment:
It would be better to remove the `foo` function itself.
```suggestion
```

Reply:
Makes sense, done.